torch_xla/experimental/xla_quantized_matmul.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn.functional as F
+import torch_xla
+from torch.library import impl
+from torch_xla.core.xla_model import XLA_LIB
+
```

Comment:
do you expect to add mores quantized ops? I recommend call this file `xla_quantized_ops` if you plan to add more ops. Having one file for every custom ops we add is not a good idea.

Reply:
I think only putting matmul related codes in this file is cleaner. For embedding, we can have another file like `xla_quantized_embedding.py`

Reply:
That makes sense, the only concern is that the file will become huge. To have user just import 1 file for all ops, we can have a `quantized_ops.py` and import all of our quantized op files there.